administrator/language/de-DE/com_users.ini

```
@@ -6,7 +6,7 @@
 
 COM_USERS="Benutzer"
 COM_USERS_A11Y_SETTINGS_FIELD_CONTRAST="Hoher Kontrast"
```

```suggestion
COM_USERS_A11Y_SETTINGS_FIELD_CONTRAST="Kontrast vergrößern"
```

---

Das finde ich besser:
```suggestion
COM_USERS_A11Y_SETTINGS_FIELD_CONTRAST="Höherer Kontrast"
```
 "kleiner/großer"  vs. "geringer/hoher" Kontrast